corehq/apps/smsbillables/management/commands/bootstrap_infobip_gateway.py

```diff
@@ -0,0 +1,35 @@
+import settings
```

Comment:
There are three new management commands in this PR with a fair amount of duplication. What do you think of combining them into a single management command with a parameter that specifies the gateway to bootstrap?

Reply:
alright, I've created a new command called "bootstrap gateway fees". Similarly, other bootstrap gateway commands could be included if needed.